src/foam/u2/view/RichChoiceView.js

```diff
@@ -439,6 +438,11 @@ foam.CLASS({
       factory: function() {
         return this.RichChoiceViewI18NComparator.create();
       }
+    },
+    {
+      class: 'String',
+      name: 'padding',
+      value: '8px 16px'
```

Comment:
Doesnt need to be a prop, should just be css, also use `0.8rem 1.6rem` rather than px